.github/workflows/test-suite.yml

```diff
@@ -53,6 +53,11 @@ jobs:
         - 5432:5432
 
     steps:
+        # eclipse-mosquitto does not support configuration via environment variables
+        # so it is not possible to use github's service feature to start the broker
+        # instead, we start the broker using docker run command and stop it at the end of the job
+      - name: "Start MQTT broker"
+        run: "docker run -d -p 1883:1883 --name mqtt eclipse-mosquitto:2.0-openssl mosquitto -c /mosquitto-no-auth.conf"
```

Comment:
Have tried to keep to a strict style within `steps` of:

```yaml
  name: "Name"
  run: scripts/<something>
```

I'd prefer not to break that if possible.